test/sys/test_mman.rs

```diff
@@ -1,11 +1,12 @@
 use nix::sys::mman::{mmap, MapFlags, ProtFlags};
+use std::num::NonZeroUsize;
 
 #[test]
 fn test_mmap_anonymous() {
     unsafe {
         let ptr = mmap(
             std::ptr::null_mut(),
-            1,
+            NonZeroUsize::new(1).unwrap(),
```

Comment:
Do you think it would be worthwhile to bring in the nonzero_ext crate just for these tests?
```suggestion
            nonzero_ext::nonzero!(1)
```

Reply:
At this point no, but if `NonZeroUsize` sees wider usage it may become worthwhile.

Reply:
BTW, I took a look through my other crates and found exactly zero places where nonzero_ext would help.  It turns out that `Option<NonZeroXXX>` is a lot more common than `NonZeroXXX`.